deployment/docker-osm-healthcare/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# This image is intended to be used as a storage
+# only container
+
+FROM byrnedo/alpine-curl
+MAINTAINER Tim Sutton <tim@kartoza.com>
+
+RUN mkdir /settings
+RUN curl -o /settings/country.pbf https://planet.openstreetmap.org/pbf/planet-latest.osm.pbf
```

Comment:
Did you test this and confirm that doing global extract works ok? What was your clip area set to? We can probably not use a clip area when the scope is global...

Reply:
hi @timlinux 
not tested yet, because i don't have space on my machine